azure-functions/acbs-function/activity-update-facility-master/index.js

```diff
@@ -44,38 +51,48 @@ const mandatoryFields = [
 ];
 
 const updateFacilityMaster = async (context) => {
-  const {
-    acbsFacilityMasterInput, updateType, facilityId, etag,
-  } = context.bindingData;
+  try {
+    const {
+      facilityId,
+      acbsFacilityMasterInput,
+      updateType,
+      etag,
+    } = context.bindingData;
 
-  const missingMandatory = findMissingMandatory(acbsFacilityMasterInput, mandatoryFields);
+    const missingMandatory = findMissingMandatory(acbsFacilityMasterInput, mandatoryFields);
 
-  if (missingMandatory.length) {
-    return Promise.resolve({ missingMandatory });
-  }
+    if (missingMandatory.length) {
+      return Promise.resolve({ missingMandatory });
```

Comment:
same as above - `await` on line 62?
otherwise we have a mix of Promise.resolve and async/await